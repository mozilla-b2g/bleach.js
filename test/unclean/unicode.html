<p>Asder ąśężźćńłó !@#$%^&*()_+{}|”:<>?,./;’[]\`~ 1234567890.</p>
<p>Hu iue ćął djd.</p>
<?><img src=1 onerror=alert(1)>
