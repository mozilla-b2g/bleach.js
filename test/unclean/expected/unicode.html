<p>Asder ąśężźćńłó !@#$%^&#38;*()_+{}|”:&#60;&#62;?,./;’[]\`~ 1234567890.</p>
<p>Hu iue ćął djd.</p>
&#60;?&#62;<img/>
